target/scripts/helpers/postfix.sh

```diff
@@ -43,21 +43,26 @@ function _vhost_collect_postfix_domains() {
   local DATABASE_VIRTUAL='/tmp/docker-mailserver/postfix-virtual.cf'
   local DOMAIN UNAME
 
-  # getting domains FROM mail accounts
+  # Extract domains from mail accounts:
   if [[ -f ${DATABASE_ACCOUNTS} ]]; then
-    while IFS=$'|' read -r LOGIN _; do
-      DOMAIN=$(echo "${LOGIN}" | cut -d @ -f2)
+    while IFS=$'|' read -r FIRST_FIELD _; do
+      # It is expected valid lines have the format local-part@domain-part:
+      DOMAIN=$(echo "${FIRST_FIELD}" | cut -d @ -f2)
+
       echo "${DOMAIN}" >>"${TMP_VHOST}"
     done < <(_get_valid_lines_from_file "${DATABASE_ACCOUNTS}")
   fi
 
-  # getting domains FROM mail aliases
+  # Extract domains from virtual alias config:
+  # Aliases may have the forms: 'local-part@domain-part', only 'local-part', or '@domain-part' (wildcard catch-all)
   if [[ -f ${DATABASE_VIRTUAL} ]]; then
-    while read -r FROM _; do
-      UNAME=$(echo "${FROM}" | cut -d @ -f1)
-      DOMAIN=$(echo "${FROM}" | cut -d @ -f2)
+    while read -r FIRST_FIELD _; do
+      UNAME=$(echo "${FIRST_FIELD}" | cut -d @ -f1)
+      DOMAIN=$(echo "${FIRST_FIELD}" | cut -d @ -f2)
```

Comment:
While `FIRST_FIELD` is absolutely correct, IMO it'd be more useful to state what the first field actually is and use this as a name. Moreover:

```suggestion
    while read -r FIRST_FIELD _; do
      UNAME=$(cut -d '@' -f 1 <<< "${FIRST_FIELD}")
      DOMAIN=$(cut -d '@' -f 2 <<< "${FIRST_FIELD}")
```